newsubprojects.md

```diff
@@ -0,0 +1,178 @@
+5. Copyright notices in individual files should be updated to the standard form described
+   in our LICENSE files.
+
+
+## Incubation of Subprojects
+
+Incubation refers to the process of a Subproject initially being developed outside the official
+Jupyter organization. Incubation is recommended for new Subprojects with the following
+characteristics:
+
+* Significant unanswered technical questions or uncertainties that require exploration.
+* Entirely new directions, scopes or ideas that haven't been vetted with the community.
+* Significant, already existing code bases where it is not clear how the Subproject will
+  integrate with the rest of Jupyter.
+
+Incubation also allows the broader community to easily distinguish between stable, officially
+supported and maintained Subprojects in the main GitHub organization and new efforts that are
+emerging and being developed for future inclusion.
+
+If there are questions about whether incubation is appropriate for a given Subproject, interested
+parties should simply start a discussion on the main Jupyter list to get community and Steering
+Council feedback.
+
+
+### Incubation in the [jupyter-incubator](https://github.com/jupyter-incubator) organization
+
+If a Subproject team knows from the start that it would eventually like its Subproject to be part
+of the official Jupyter organization, it can be incubated in the
+[jupyter-incubator](https://github.com/jupyter-incubator) organization.
+
+The goals of the incubation in the [jupyter-incubator](https://github.com/jupyter-incubator) organization are as follows:
+
+* Contributors can quickly and easily get their code exposed to the Jupyter community while
+  complying with individual and organizational contribution restrictions.
+* Contributors can work with the community and Steering Council and gather feedback early and often
+  that will help them develop and refine a clear and concise integration proposal.
+* Allow the community to distinguish between officially supported Subprojects and experimental
+  Subprojects pursued by members of the community.
+
+### Proposal for incubation
+
+A Subproject team can initiate the [jupyter-incubator](https://github.com/jupyter-incubator)
+incubation process by submitting a proposal for incubation. This process is designed to be
+lightweight and quick:
+
+1. The proposers must fill out an incubation application by sumitting a pull request to the
+   [jupyter-incubator/proposals](https://github.com/jupyter-incubator/proposals) repository.
+2. Proposers must announce their intent to the community with a post to the main Jupyter
+   mailing list.
+3. The proposed Subproject must have an Advocate who is an active Steering Council member. The role
+   of the Advocate is to introduce the Subproject team to the Jupyter community and help them through
+   incubation processes. While the Advocate may be involved in the actual development and design
```

Comment:
I like having them help through more than the application process. I added the following sentence to clarify that doesn't mean they will code. But I think in reality, Advocate's will help in small ways even after the proposal phase. We can revisit this later as well...